docs/operations/vm_rollout_strategies.md

```diff
@@ -0,0 +1,67 @@
+In other words, it defines when and how changes to a VM object get propagated to its corresponding VMI object.
+
+There are currently 2 rollout strategies: LiveUpdate and Stage.
+Only 1 can be specified and the default is Stage.
+
+## Feature Gate
+
+As long as the `VMLiveUpdateFeatures` is not enabled, the VM Rollout Strategy is ignored and defaults to "Stage".
+The feature gate is set in the KubeVirt custom resource (CR) like that:
+
+```yaml
+apiVersion: kubevirt.io/v1
+kind: KubeVirt
+spec:
+  configuration:
+    developerConfiguration:
+      featureGates:
+        - VMLiveUpdateFeatures
+```
+
+## LiveUpdate
+
+The `liveUpdate` VM rollout strategy tries to propagate VM object changes to running VMIs as soon as possible.  
+For example, changing the number of CPU sockets will trigger a [CPU hotplug](./cpu_hotplug.md).
+
+This is how the strategy is set in the KubeVirt CR:
+
+```yaml
+apiVersion: kubevirt.io/v1
+kind: KubeVirt
+spec:
+  configuration:
+    vmRolloutStrategy:
+      liveUpdate: {}
+```
+
+## Stage
+
+The `stage` VM rollout strategy on the other end stages every change made to the VM object until its next reboot.  
+
+```yaml
+apiVersion: kubevirt.io/v1
+kind: KubeVirt
+spec:
+  configuration:
+    vmRolloutStrategy:
+      stage: {}
+```
+
+## RestartRequired condition
+
+Any change made to a VM object when the rollout strategy is `stage` will trigger the `RestartRequired` VM condition.  
+When the rollout strategy is `liveUpdate`, only non-propagatable changes will trigger the condition.
+
+Once the RestartRequired condition is set on a VM object, not further change can be propagated, even when the strategy is `liveUpdate`.  
+Changes will become effective on next reboot, and the condition will be removed.
+
+## Limitations
+
+The current implementation has the following limitations:
+- To track generic changes to VM objects, we keep a copy of the previous generation as a [controller revision](https://kubernetes.io/docs/reference/kubernetes-api/workload-resources/controller-revision-v1/). That slightly increases VM footprints and adds some overhead to the VM controller's main loop .
+- Once the `RestartRequired` condition is set, the only way to get rid of it is to restart the VM. In the future, we plan on implementing a way to get rid of it by revering the VM template spec to its last non-RestartRequired state.
+- Cluster defaults are excluded from this logic. It means that changing a cluster-wide setting that impacts VM specs will not be live-updated, regardless of the rollout strategy.
+- The `RestartRequired` condition comes with a message stating what kind of change triggered the condition (CPU/memory/other). That message pertains only to the first change that triggered the condition. Additional changes that would usually trigger the condition will just get staged and no additional `RestartRequired` condition will be added.
```

Comment:
The output format is wrong. It is not shown as a list https://deploy-preview-758--kubevirt-user-guide.netlify.app/operations/vm_rollout_strategies/